src/Psalm/Baseline.php

```diff
@@ -0,0 +1,159 @@
+<?php
+namespace Psalm;
+
+class Baseline
```

Comment:
One last thing - I have a slight feeling the `ErrorBaseline` might be better. My I don't have any firm reason for that, and maybe `Baseline` is obvious enough.